### ISSUE TITLE
Authorization tests should use role-level fixtures, not `auth_headers`

### ISSUE THREAD

**Issue by aedwardg** [codeforpdx/dwellingly-app#547]
### Purpose
**Step 3 of 3** in moving away from using `auth_headers` in the integration tests
This should take place **after** #545

Authorization tests are for testing whether a user has the role/permissions to perform a certain action.  To do this, we don't need to hit the login endpoint every time (which is what `auth_headers` is doing).   For more information, see #545 

### Proposed Solution
Replace the `auth_headers` fixture with with the appropriate role-level fixture (`admin_header`, `staff_header`, `pm_header`) in all integration tests that are testing authorization. Once #545 is complete, these should be in a separate directory.  
NOTE:
- This may require altering the tests to use `empty_test_db` as well if the test doesn't already.
- There is no header fixture for "pending" users, because these users have no role and will not be authenticated.  To test that a pending user cannot perform an action, write the API call with no header information.

### Acceptance Criteria
Please list the requirements the implementation of this feature should meet.
- [ ] All integration tests that test _authorization_ (i.e., checks if the user has permission to perform the action) should use the appropriate role-level fixture instead of `auth_headers`.
- [ ] All tests should still pass.

**Comment by NickSchimek** [comment]
This seems to be a pretty broadly scoped issue. @ovsyankimo Do you feel ok taking this on or can I break it up into smaller issues?

**Comment by ovsyankimo** [comment]
> This seems to be a pretty broadly scoped issue. @ovsyankimo Do you feel ok taking this on or can I break it up into smaller issues?

@NickSchimek I feel like I could take it on BUT it might take me forever. If it's better to close tickets quickly then more, smaller tickets would be better.

**Comment by aedwardg** [comment]
Closing in favor of issues #628, #629, #630, #631, #632, #633, #634, & #635
